update/config.js

```diff
@@ -1,13 +1,15 @@
-const config = require('./config.json' );
+
+var fs = require('fs');
+require('dotenv').config()
+
+var configured = false;
+
+if (fs.existsSync('.env')) {
+  configured = true;
+} else {
+  console.log(".env file not found")
```

Comment:
i think at this point we may want to read from environment variables, but that can be done later.